README.md

```diff
@@ -0,0 +1,45 @@
+This repository is for the Workflow CA.
+
```

Comment:
Well done on the project setup and the addition of badges. PLs, you need to add some info about the new configuration you added such as how to run the test script.